### PR TITLE
IBX-1868: Added information about reference and supplement the missing information 

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [IBX-1868](https://issues.ibexa.co/browse/IBX-1868)
| Versions      | v3.3, v4

### CODE REVIEW

File: docs/guide/data_migration/data_migration_actions.md [hunk 1]
```diff
@@ -159,3 +157,17 @@ mode: Create and Update
...
+mode: Update
+``` yaml
+    actions:
+        -
+            action: unassign_role_user_group
+            id: 1
+```
+**Note**
+
+In action `unassign_role_user_group` id is role assignment ID from table `ezuser_role`
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration_actions.md:173]:
```suggestion
!!! note
    
    In the `assign_user_group_to_role` action, Limitation type Section can only use Section ID:
    
    mode: Update
    
    ``` yaml
        actions:
            -
                action: unassign_role_user_group
                id: 1
    ```
    
    In the `unassign_role_user_group` action, the ID is Role assignment ID from the `ezuser_role` table.
```

File: docs/guide/data_migration/data_migration.md [hunk 2]
```diff
@@ -18,3 +18,9 @@ between installations by using the migration command.
     
     As of v3.3.3, the `ezmigrationbundle` package has been removed to mitigate this issue. 
     It is recommended that you use the default `ibexa/migrations` package to migrate your data. 
+
+
+## Examples of using migration
+
+You can find examples of using migrations in your project's
+`vendor/ibexa/migrations/tests/bundle/Command/MigrateCommand/migrate-command-fixtures` folder.
```
Comment (DominikaK) [inline docs/guide/data_migration/data_migration.md:26]:
```suggestion
You can find examples of using migrations in [Migration file content](exporting_and_importing_data.md#migration-file-content),
and in your project's `vendor/ibexa/migrations/tests/bundle/Command/MigrateCommand/migrate-command-fixtures` folder.
```
I suggest adding this as well, as started in https://github.com/ezsystems/developer-documentation/pull/1545/files#diff-da96e7f993350f34c0356c9d6b58ccd98bab222096cbc50dd49ce445d7407095R267

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/guide/data_migration/data_migration_actions.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/data_migration/data_migration_actions.md]: The note content needs to be indented for the formatting to work, for now it's broken: 
image

Comment (mateuszdebinski) [reply docs/guide/data_migration/data_migration_actions.md]: now should be correct

**Review by DominikaK — APPROVED** [review]

**Pull request merged**